scripts/sct_deepseg.py

```diff
@@ -16,6 +16,9 @@
 import sys
 
 import spinalcordtoolbox.deepseg as deepseg
+import spinalcordtoolbox.deepseg.models
+import spinalcordtoolbox.deepseg.core
+
```

Comment:
Just to be clear about the PR description, this change is new, and wasn't a part of f65df321b078b939e5de66755ca0561fe773ed82, right?

From what I understand, though, this is needed to fix the usage of `deepseg.models.TASKS` and `deepseg.core.DEFAULTS`. So, it looks OK to me.

Reply:
Yes this was from another commit which I didn't bother to track and revert.